source/usage-examples/struct-tagging.txt

```diff
@@ -0,0 +1,69 @@
+===========================
+Set Custom BSON Field Names
+===========================
+
+.. default-domain:: mongodb
+
+You can customize the way that the Go Driver converts Go
+structs to :manual:`BSON </reference/bson-types/>` using 
+**struct tags**, which are optional instructions you can use with struct
+fields.
```

Comment:
Question: Should a struct tag be defined here? It seems the format of our other usage examples pages don't contain any definition.

Reply:
okay, will omit

Reply:
since it's no longer defined, you can unbold the term too